meta/FSMathLink.m

```diff
@@ -393,14 +512,16 @@
 
       {
          Redirect_output crd(link);
-         auto setting = data.get_settings();
-         if (!static_cast<bool>(setting.get(flexiblesusy::Spectrum_generator_settings::calculate_sm_masses)) ||
-            !static_cast<bool>(setting.get(flexiblesusy::Spectrum_generator_settings::calculate_bsm_masses))) {
+         auto settings = data.get_settings();
+         auto fdSettings = data.get_fd_settings();
+         if (!static_cast<bool>(settings.get(flexiblesusy::Spectrum_generator_settings::calculate_sm_masses)) ||
```

Comment:
See my proposal to avoid the `static_cast<bool>` and write
~~~.cpp
if (settings.get(...) != 0) { ... }
~~~